validator.js

```diff
@@ -2420,6 +2524,7 @@ var validator = {
   isJSON: isJSON,
   isEmpty: isEmpty,
   isLength: isLength,
+  isLocale: isLocale,
```

Comment:
@profnandaa Should this be removed as well ?

Reply:
Yes, please -- all the unrelated diff.